sdcm/nemesis.py

```diff
@@ -6458,3 +6536,22 @@ class DisableBinaryGossipExecuteMajorCompaction(Nemesis):
 
     def disrupt(self):
         self.disrupt_disable_binary_gossip_execute_major_compaction()
+
+
+class IsolateNodeWithProcessSignalNemesis(Nemesis):
+    disruptive = True
+    topology_changes = True
+    kubernetes = False
+
+    def disrupt(self):
+        self.disrupt_refusing_connection_from_banned_node(use_iptables=False)
+
+
+class IsolateNodeWithIptableRuleNemesis(Nemesis):
+    disruptive = True
+    topology_changes = True
+    kubernetes = False
+    free_tier_set = True
```

Comment:
remove

Reply:
@roydahan , these lines should be remove? and file sdcm/nemesis.py shouldn't be updated?

Reply:
Only the "free_tier_set", I don't think it's relevant for this nemesis.

Reply:
fixed